src/teb_config.cpp

```diff
@@ -350,6 +352,13 @@ void TebConfig::checkParameters() const
   // weights
   if (optim.weight_optimaltime <= 0)
       ROS_WARN("TebLocalPlannerROS() Param Warning: parameter weight_optimaltime shoud be > 0 (even if weight_shortest_path is in use)");
+
+  // holonomic check
+  if (robot.max_vel_y >= 0) {
```

Comment:
```suggestion
  if (robot.max_vel_y > 0) {
```
F: it's 0 by default, so this check always happens.